packages/friends/yml/en/samuel-neale.yml

```diff
@@ -7,9 +7,10 @@ description:
   ministry of Catharine Payton and Mary Peisley (his future wife) while they were
   ministering in the city of Dublin, Ireland. Having seen and felt that he was traveling
   on the broad path that leads to destruction, Samuel Neale set his heart to "wait upon
-  the Lord in the way of His judgments." And becoming acquainted with the true baptism of
-  Christ (that of Spirit and fire), he was called into the ministry, and spent much of his
-  life traveling and ministering to the Lord's body.
+  the Lord in the way of His judgments." And having become acquainted with the true
```

Comment:
I think I prefer leaving off the `And` here, and making it just `Having become acquainted...`.